.gitlab-ci.yml

```diff
@@ -115,14 +115,13 @@ upload-artifacts:
     # on the command line.
     - ln -s "$CI_OWNCLOUD_PASSWORD" ~/.netrc
     - sed --follow-symlinks -i 's/^/machine ftp.essensium.com login prplmesh-robot-ci password /' ~/.netrc
-    - ci/owncloud/upload_to_owncloud.sh -v "artifacts/$CI_COMMIT_SHA-$CI_JOB_ID" ./build
     - |
       if [ $CI_COMMIT_BRANCH = "master" ] ; then
```

Comment:
(nitpick) this could be replaced with an `only: master`. Except if we want to keep this job around just to collect all the artefacts in a single job.